### PR TITLE
Prepare 0.70.0 release

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -4,6 +4,33 @@ History
 -------
 
 .. to_doc
+* Temporarily add galaxy-util requirement `Pull Request 991`_
+* Make symlinks in tool tree work for planemo test `Pull Request 988`_
+* Reduce use of `shell=True` in subprocesses `Pull Request 989`_
+* Drop planemo database seed option `Pull Request 985`_
+* Don't execute `untar_to()` subprocesses through the shell `Pull Request
```
Comment (nsoranzo) [inline HISTORY.rst]: Cannot break the line here (also at lines 18-19 and 27-28).

File: Makefile [hunk 2]
```diff
@@ -77,7 +77,7 @@ lint: ## check style using tox and flake8 for Python 2 and Python 3
 	$(IN_VENV) tox -e py27-lint && tox -e py37-lint
 
 lint-readme: ## check README formatting for PyPI
```
Comment (nsoranzo) [inline Makefile]: Rename to `check-sdist` ?

File: HISTORY.rst [hunk 3]
```diff
@@ -6,10 +6,29 @@ History
...
-0.63.0.dev0
----------------------
-
-    
+0.70.0 (2020-01-29)
+---------------------
+
+* Temporarily add galaxy-util requirement `Pull Request 991`_
+* Make symlinks in tool tree work for planemo test `Pull Request 988`_
+* Reduce use of `shell=True` in subprocesses `Pull Request 989`_
```
Comment (nsoranzo) [inline HISTORY.rst]:
```suggestion
* Reduce use of ``shell=True`` in subprocesses `Pull Request 989`_
```

File: HISTORY.rst [hunk 4]
```diff
@@ -6,10 +6,29 @@ History
...
+* Drop planemo database seed option `Pull Request 985`_
+* Don't execute `untar_to()` subprocesses through the shell `Pull Request  984`_
```
Comment (nsoranzo) [inline HISTORY.rst]:
```suggestion
* Don't execute ``untar_to()`` subprocesses through the shell `Pull Request  984`_
```

File: HISTORY.rst [hunk 5]
```diff
@@ -6,10 +6,29 @@ History
...
+* Allow setting database_connection for planemo test runs `Pull Request 986`_
+* Fix copy-paste mistakes `Pull Request 983`_
+* Add planemo list_repos command `Pull Request 982`_
+* Make container_register build files with headers and include base_image `Pull Request 980`_
+* Replace deprecated galaxy-lib requirement with galaxy-tool-util `Pull  Request 978`_
+* close all opended files (thanks to `@bernt-matthias`_). `Pull Request 979`_
```
Comment (nsoranzo) [inline HISTORY.rst]:
```suggestion
* Close all opened files (thanks to `@bernt-matthias`_). `Pull Request 979`_
```

File: HISTORY.rst [hunk 6]
```diff
@@ -6,10 +6,29 @@ History
...
+* Build single requirement container, log if requirement not in best-pr… `Pull Request 977`_
```
Comment (nsoranzo) [inline HISTORY.rst]:
```suggestion
* Build single requirement container, log if requirement not in best-practice channels `Pull Request 977`_
```

File: Makefile (hunk 2 above)
Comment (mvdbeek) [reply Makefile]: Good idea, done

**Pull request merged**